source/tutorials.rst

```diff
@@ -107,6 +187,12 @@ The WEC-Sim Simulink model is created by dragging and dropping blocks from the *
 
 	When setting up a WEC-Sim model, it is very important to note the base and follower frames.
 
+.. warning::
+    A bit more detail about connecting the blocks would be useful, particularly
+    with the note above. The tutorial doesn't really make clear what base
+    and follower frame means (nor links to an explanation), nor does it say
+    that the connectors are marked with a "B" and an "F" to indicate which is
+    which. Even in the diagrams, the blocks are too small to see this.
```

Comment:
This is a Simulink question, we could add a reference to Simulink

Reply:
I think the descriptions of the frames have been improved a lot since I wrote this.